internal/stacks/stack_manager.go

```diff
@@ -558,6 +575,11 @@ func (s *StackManager) PullStack(verbose bool, options *types.PullOptions) error
 		images = append(images, constants.SandboxImageName)
 	}
 
+	// Also pull the FFTM if we're using it
+	// if s.Stack.FFTMEnabled {
+	// 	images = append(images, constants.FFTMImageName)
+	// }
+
```

Comment:
Is this needed?

Reply:
Yes, now we have images in ghcr. This gets to the point of not having a manifest - as if you want to use local images for these things, you have to do source code edits like this at the moment to avoid the auto-pulls (which fail).